web/app.js

```diff
@@ -224,7 +224,10 @@ const PDFViewerApplication = {
     } catch (ex) {
       console.error(`initialize: "${ex.message}".`);
     }
-    if (AppOptions.get("pdfBugEnabled")) {
+    if (
+      (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) ||
+      AppOptions.get("pdfBugEnabled")
+    ) {
```

Comment:
Could we just change https://github.com/mozilla/pdf.js/blob/130a0fef3d96d28c50dd40d89cf75b2d1ed12e24/web/app_options.js#L206 as follows instead?
```javascript
value typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING"),
```